### PR TITLE
Adds rpc param

### LINKED ISSUE #7: Add custom RPC parameter
For context, this [post](https://near.social/mob.near/widget/MainPage.N.Post.Page?accountId=efiz.near&blockHeight=115257294) and then [comment](https://near.social/mob.near/widget/MainPage.N.Comment.Page?accountId=petersalomonsen.near&blockHeight=115381935)

Fast Near was enabled on near.social gateway via this [change here](https://github.com/NearSocial/viewer/commit/cf7c43376ab4d247af42576b2cbabf38f1ac9f4b), line 97 -- Could we pass RPC the same way here?

### PR DESCRIPTION
**Pull request by elliotBraem**
Continues discussion from [here](https://github.com/NEARBuilders/near-bos-webcomponent/pull/8).

Resolves #7 

Adds tests:
- "Verify default RPC is called when value not provided" (rpc attribute is not set)
- "Verify custom RPC is called when provided" (rpc attribute is set)

To test with local rpc, just run a workspace with the most recent bos-workspace installed and use the proxy url from the output:

Screenshot 2024-04-30 at 5 08 05 PM

If you want to make changes to bos-workspace, run [examples/single](https://github.com/NEARBuilders/bos-workspace/tree/main/examples) with a local build -- the proxy happens in [lib/server](https://github.com/NEARBuilders/bos-workspace/blob/fef1dd512546b4a7ce7ac223dc4bf3ee0dcbf6d5/lib/server.ts#L84). Make sure you have turn off Brave shields.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -1,107 +1,117 @@
...
+    />
+  </head>
+  <body>
+    <div>
+      <button id="open-walletselector-button" type="button">
+        Open wallet selector
+      </button>
+    </div>
+    <near-social-viewer></near-social-viewer>
+  </body>
```
Comment (elliotBraem) [inline public/index.html:20]: (linting, no other change here)

File: src/App.scss [hunk 2]
```diff
@@ -32,18 +32,3 @@ a {
   }
 }
 
-.sidebar-items {
```
Comment (elliotBraem) [inline src/App.scss:35]: unused css

**Review by petersalomonsen — APPROVED** [review]
Looking good 👍

File: config/webpack.development.js [hunk 3]
```diff
@@ -2,7 +2,7 @@ const path = require("path");
 const { HotModuleReplacementPlugin } = require("webpack");
 
 module.exports = () => ({
-  devtool: false,
+  devtool: 'inline-source-map',
```
Comment (klngrs) [inline config/webpack.development.js:5]: Nice - I feel like we should have this in all webpack repos

**Review by klngrs — APPROVED** [review]
Looks good to me

**Pull request merged**